lua/telescope/pickers/entry_display.lua

```diff
@@ -43,7 +43,7 @@ end
 
 local function truncate(str, len)
   -- TODO: This doesn't handle multi byte chars...
-  if vim.fn.strdisplaywidth(str) > len - 1 then
+  if vim.fn.strdisplaywidth(str) > len then
```

Comment:
Let me explain this one quick.
If i have a perfect match, string is 3 and width is 3 it will still be truncated.

Reply:
Makes sense to me